internal/operands/node-labeller/resources.go

```diff
@@ -106,12 +84,28 @@ func newClusterRoleBinding(namespace string) *rbac.ClusterRoleBinding {
 }
 
 func newConfigMap(namespace string) *core.ConfigMap {
+	var cpuPluginConfigmap = `obsoleteCPUs:
```

Comment:
This could be a const I think.
Then it wouldn't even need to be in there but could remain global.